common/custom-sidebar.js

```diff
@@ -85,6 +85,21 @@ export function updateSidebar(vizConfiguration) {
 	};
   window.viewer.scene.addEventListener("truth_layer_added", onTruthLayerAdded);
 
+  const onTruthLayerDeleted = (e) => {
+    const truthNodes = tree.jstree("get_children_dom", truthVizTree);
+    let nodeID;
+    for (let i = 0; i < truthNodes.length; i++) {
+      if (truthNodes[i].textContent === e.truthLayer.name) {
+        nodeID = truthNodes[i].id;
+        break;
+      }
+    }
```

Comment:
Please use `findIndex`.

Reply:
Note that it will return `i`, not `nodeID`.

Reply:
`findIndex` gives me an error. I don't believe it's supported for whatever data structure `truthNodes` is. I had tried to use `find` before I used the for loop, but that didn't quite work either.